Makefile

```diff
@@ -349,3 +349,27 @@ integration-test:
 	else\
 		$(GINKGO) -v $(TEST_ARGS) --fail-fast --focus-file=$(TEST_FILE) test/integration;\
 	fi
+
+#hosted
+
+.PHONY: bootstrap-hosted
+bootstrap-hosted: clone-hosted install-hosted copy-config install-crds install-resources kind-deploy-policy-framework kind-deploy-policy-controllers
+
+.PHONY: install-hosted
+install-hosted: 
+	@cd ./governance-policy-addon-controller && KIND_VERSION=latest HOSTED_MODE=true ./build/manage-clusters.sh
```

Comment:
Instead of hardcoding `KIND_VERSION=latest`, could you please use the value coming from `KIND_VERSION` in this Makefile?